nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -62,23 +63,33 @@ def _ml_parametrization(
         elif regressor == "mlp":
             regr = MLPRegressor(alpha=alpha, activation=activation, solver=solver,
                                 learning_rate=learning_rate, random_state=0)
+        elif regressor == "kerasDenseNN":
+            regr = keras.Sequential([
+                    keras.layers.Dense(64, activation=activation, input_shape=(self.X_train.shape[1],)),
+                    keras.layers.Dense(1)
+            ])
+            regr.compile(optimizer=solver, loss='mse', metrics=['mse'])
         else:
             raise ValueError(f"Unknown regressor {regressor}.")
 
         if noise_free:  # noise_free is True when we want the result on the test set.
-            X = self.X_train
-            y = self.y_train
             X_test = self.X_test
             y_test = self.y_test
-            regr.fit(np.asarray(self.X), np.asarray(self.y))
+            if regressor == "kerasDenseNN":
+                regr.fit(np.asarray(self.X_train), np.asarray(self.y_train), epochs=20)
```

Comment:
why do they need to be cast as array? Can't they be arrays from the start? That would simplify the code, and according the the type hints, they should be

Reply:
you're right.